### PR TITLE
3 update the header icon

### LINKED ISSUE #3: Update the header icon
Please add a rounded header to the title section of the doc

### PR DESCRIPTION
**Pull request by ConradKash**
Please Approve the changes to the logo

### CODE REVIEW

**Comment by ConradKash** [comment]
Hey Beth please Approve this @Symwe

File: index.html
Comment (Symwe) [inline index.html:1]: The simplicity of the logo ensures easy recognition and scalability across various digital and print platforms.

Comment (Symwe) [inline index.html:1]: The simplicity of the logo ensures easy recognition and scalability across various digital and print platforms.

Comment (ConradKash) [reply index.html:1]: Asht. First Review

Comment (ConradKash) [reply index.html:1]: Yes I understand

File: assets/img/62792288.png
Comment (Symwe) [inline assets/img/62792288.png:1]: The HTML codebase demonstrates a clear and structured organization, facilitating easy maintenance and future modifications for the website.
Proper implementation of meta tags and alt attributes enhances the website's SEO, enabling better visibility and indexing by search engines.

Comment (ConradKash) [reply assets/img/62792288.png:1]: HAhaha

**Review by Symwe — APPROVED** [review]
Clear

**Pull request merged**